src/models/CourseModel.js

```diff
@@ -0,0 +1,103 @@
+const admin = require('firebase-admin');
+
+class CourseModel {
+  constructor({
+    coursesCode,
+    courseName,
+    typeOfCourse,
+    description,
+    courseImgUrl,
+    logoUrl,
+    points,
+    rating,
+    otherInfo,
+    videoURL = [], // Default to empty array if not provided
+    lessonInfo = {}, // Default to empty object if not provided
+    courseTutorialImgURL = [], // Default to empty array if not provided
+    quizCode = [], // Default to empty array if not provided
+    feedback,
+    createdBy,
```

Comment:
I think this should be "owner" and be the id of the teacher that creates it.

Reply:
I will discuss with team and will changes course collection fields.